dashboard/lib/pd/payment/payment_factory.rb

```diff
@@ -27,7 +27,7 @@ def self.get_calculator_class(workshop)
         ].include?(workshop.course)
       end
 
-      nil
+      PaymentCalculatorUnpaid
```

Comment:
Added a test [workshops that do not match a known payment type fall through to unpaid](https://github.com/code-dot-org/code-dot-org/pull/12651/commits/0ba995df1f7d0b6986382431d80828be04399aaf#diff-2b6b6c2c760761d8b2e4668eae8dc74eR67)